collectors/nvd/collectors.py

```diff
@@ -248,7 +178,7 @@ def collect(self, cve: Union[str, None] = None) -> str:
             try:
                 flaw = Flaw.objects.get(cve_id=cve_id)
 
-                # update NVD CVSS2 or CVSS3 data if necessary
+                # update NVD CVSS2, CVSS3 and CVSS4 data if necessary
                 if self.update_cvss_via_flawcvss(flaw, item):
                     updated_flaws.append(cve_id)
                     # no automatic timestamps as those go from Bugzilla
```

Comment:
I am not sure if this is still true because we made several Bugzilla changes.

Reply:
Totally not true any more :smile: no flaw timestamps go from Bugzilla now

Reply:
Okay, I will change it accordingly :)